login.php

```diff
@@ -54,7 +54,7 @@
         <div class="input-group mb-3">
           <input type="email" class="form-control" name="email" required placeholder="Email">
           <div class="input-group-append">
-            <div class="input-group-text">
+            <div class="input-group-text" id="password-toggle">
```

Comment:
This won't work - you are adding the id to the wrong icon and It is not being made a clickable attribute 
After making changes - please add screenshots to the PR